safeeyes/safeeyes.py

```diff
@@ -63,12 +63,24 @@ def __init__(self, system_locale, config, cli_args):
         self.rpc_server = None
         self._status = ''
         self.cli_args = cli_args
+        self.system_locale = system_locale
+
+    def start(self):
+        """
+        Start Safe Eyes
+        """
+        self.run()
+
+    def do_startup(self):
+        Gtk.Application.do_startup(self)
+
+        logging.info('Starting up Application')
 
         # Initialize the Safe Eyes Context
         self.context['version'] = SAFE_EYES_VERSION
         self.context['desktop'] = utility.desktop_environment()
         self.context['is_wayland'] = utility.is_wayland()
-        self.context['locale'] = system_locale
+        self.context['locale'] = self.system_locale
```

Comment:
Why is it necessary to first define another extra variable and then reuse it?

Reply:
This was moved out of the constructor, and into a separate method.
In theory I could start filling `self.context` in the constructor instead, but I'd prefer keeping it in one place.